handle_links/explainer.md

```diff
@@ -40,17 +40,14 @@ In a similar vein, a user can click on a link of a music streaming service (www.
 We propose the addition of the `handle_links` member to the web app manifest that specifies the default link handling for the installed web app. The shape of this member is as follows:
 
 ```json
-"handle_links": {
-    "mode" : "auto" | "all" | "none"
-}
+"handle_links": "auto" | "preferred" | "not-preferred"
 ```
 
-* `all`: the user agent should open in-scope links within the installed application.
-* `none`: the use agent must not open links within the installed applications.
+* `preferred`: the user agent should open in-scope links within the installed application.
+* `not-preferred`: the use agent must not open links within the installed applications.
```

Comment:
*user

The "must not" should probably change.

Reply:
Fixed spelling. Changed "must" to "should".